projects/plugins/backup/src/js/components/Backups.js

```diff
@@ -0,0 +1,351 @@
+								filterUnits="userSpaceOnUse"
+								color-interpolation-filters="sRGB"
+							>
+								<feFlood flood-opacity="0" result="BackgroundImageFix"></feFlood>
+								<feColorMatrix
+									in="SourceAlpha"
+									values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"
+								></feColorMatrix>
+								<feOffset></feOffset>
+								<feGaussianBlur stdDeviation="20"></feGaussianBlur>
+								<feColorMatrix values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.08 0"></feColorMatrix>
+								<feBlend in2="BackgroundImageFix" result="effect3_dropShadow"></feBlend>
+								<feBlend in="SourceGraphic" in2="effect3_dropShadow" result="shape"></feBlend>
+							</filter>
+						</defs>
+					</svg>
+				</div>
+			</div>
+		);
+	};
+
+	const formatDateString = dateString => {
+		const todayString = __( 'Today', 'jetpack-backup' );
+		const todayDate = getDate();
+		let backupDate = todayString;
+		if ( dateI18n( 'zY', todayDate ) !== dateI18n( 'zY', dateString ) ) {
+			backupDate = dateI18n( 'M j', dateString );
+		}
+		const backupTime = dateI18n( 'g:i A', dateString );
+
+		return backupDate + ', ' + backupTime;
+	};
+
+	const renderCompleteBackup = () => {
+		return (
+			<div className="jp-row">
+				<div className="lg-col-span-3 md-col-span-4 sm-col-span-4">
+					<div className="backup__latest">
+						<img src={ CloudIcon } alt="Jetpack Logo" />
+						<h2>{ __( 'Latest Backup', 'jetpack-backup' ) }</h2>
+					</div>
+					<h1>{ formatDateString( latestTime ) }</h1>
+					<a
+						class="button is-full-width"
+						href={ 'https://cloud.jetpack.com/backup/' + domain }
+						target="_blank"
+						rel="noreferrer"
+					>
+						{ __( 'See all your backups', 'jetpack-backup' ) }
+					</a>
+				</div>
+				<div className="lg-col-span-1 md-col-span-4 sm-col-span-0"></div>
+				<div className="lg-col-span-2 md-col-span-2 sm-col-span-2">
+					<StatBlock icon={ PostsIcon } label="Posts" value={ stats.posts } />
+				</div>
+				<div className="lg-col-span-2 md-col-span-2 sm-col-span-2">
+					<StatBlock icon={ UploadsIcon } label="Uploads" value={ stats.uploads } />
+				</div>
+				<div className="lg-col-span-2 md-col-span-2 sm-col-span-2">
+					<StatBlock icon={ PluginsIcon } label="Plugins" value={ stats.plugins } />
+				</div>
+				<div className="lg-col-span-2 md-col-span-2 sm-col-span-2">
+					<StatBlock icon={ ThemesIcon } label="Themes" value={ stats.themes } />
+				</div>
+			</div>
+		);
+	};
+
+	const renderNoBackups = () => {
+		return (
+			<div class="jp-row">
+				<div class="lg-col-span-5 md-col-span-4 sm-col-span-4">
+					<h1>{ __( "We're having trouble backing up your site", 'jetpack-backup' ) }</h1>
+					<p>
+						{ __( 'Check that your Jetpack Connection is healthy with the ', 'jepack-backup' ) }
+						<a
+							href={ 'https://tools.jetpack.com/debug/?url=' + domain }
+							target="_blank"
+							rel="noreferrer"
+						>
+							{ __( 'Jetpack Debugger', 'jetpack-backup' ) }
+						</a>
+						.
+					</p>
```

Comment:
This will be hard to translate in other langages since the strings are split.

I would recommend relying on `@wordpress/element` and its `createInterpolateElement` to build a full string with a React element. Something like this:

```js
{ createInterpolateElement(
	__(
		'Check that your Jetpack Connection is healthy with the <a>Jetpack Debugger</a>.',
		'jetpack-backup'
	),
	{
		a: (
			<a
				href={ 'https://tools.jetpack.com/debug/?url=' + domain }
				target="_blank"
				rel="noreferrer"
			/>
		),
	}
) }
```

You may even be able to go one step further and rely on `ExternalLink` from `@wordpress/components` here, since that's an external link:

```js
{ createInterpolateElement(
	__(
		'Check that your Jetpack Connection is healthy with the <a>Jetpack Debugger</a>.',
		'jetpack-backup'
	),
	{
		a: <ExternalLink href={ `https://tools.jetpack.com/debug/?url=${ domain }` } />,
	}
) }
```

I think `createInterpolateElement` could also be useful in other split strings you have in this PR.

Reply:
I updated to use createInterpolateElement.  ExternalLink comes with some additional styling / the external link icon so I didn't incorporate that.  It may make sense in most places, but I didn't want to impact the design.  If the svg becomes optional I could use it in more places, including the `See all your backups` buttons.